client/my-sites/stats/stats-detail-months/index.jsx

```diff
@@ -130,13 +130,10 @@ export default React.createClass( {
 				</div>
 				<div className="module-content">
 					<div className="module-content-text module-content-text-info">
```

Comment:
Good call @timmyc. Updated in https://github.com/Automattic/wp-calypso/commit/fcf284d3c951188542acfcdf45613dc66d51dc9f